R/data.table.R

```diff
@@ -1739,21 +1740,36 @@ replace_dot_alias = function(e) {
       } else {
         # Apply GForce
         .gforce_ok = function(q) {
```

Comment:
nit: this function is getting pretty unwieldy... it should be moved out as a helper and then we can test it more directly.

feel free to add a TODO for now.

Reply:
comment added